tetris/piece.go

```diff
@@ -325,3 +370,7 @@ func (p *Piece) bounds(w int) {
 		}
 	}
 }
+
+func (p *Piece) GetColor() string {
```

Comment:
just Color(), drop the Get. 

Also move exported function up so that they show before unexported ones in the file.